README.md

```diff
@@ -169,7 +169,9 @@ When you run `azd up` after and are prompted to select a value for `openAiResour
   then run `azd env set AZURE_SEARCH_SERVICE_LOCATION {Location of existing service}`
 1. If the search service's SKU is not standard, then run `azd env set AZURE_SEARCH_SERVICE_SKU {Name of SKU}`. The free tier won't work as it doesn't support managed identity. ([See other possible values](https://learn.microsoft.com/azure/templates/microsoft.search/searchservices?pivots=deployment-language-bicep#sku))
 1. If you have an existing index that is set up with all the expected fields, then run `azd env set AZURE_SEARCH_INDEX {Name of existing index}`. Otherwise, the `azd up` command will create a new index.
-
+1. If you want to configure the language of the search query, then run `azd env set AZURE_SEARCH_QUERY_LANGUAGE {Name of query language}`. ([See other possible values](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/search/azure-search-documents/azure/search/documents/aio/_search_client_async.py#L226))
```

Comment:
I was going to suggest linking to the generated reference doc at https://learn.microsoft.com/en-us/python/api/azure-search-documents/azure.search.documents.aio.searchclient?view=azure-python#azure-search-documents-aio-searchclient-search but it doesnt show those parameters. @mattmsft Do you know why theyre missing?

Reply:
I do not know. But we can link to this which also has all the types
https://learn.microsoft.com/en-us/python/api/azure-search-documents/azure.search.documents.models.querylanguage?view=azure-python-preview

Reply:
Thanks for pointing it out @pamelafox @mattmsft 
It is exactly where I'm not confident.
I updated links [923a94e](https://github.com/Azure-Samples/azure-search-openai-demo/pull/780/commits/923a94ec15000dc13367e5324a028998bcab0ac8)
But unfortunately, there's no good reference for analyzer types in Python sdk docs, so I refer to dotnet sdk ones. 
https://github.com/Azure-Samples/azure-search-openai-demo/pull/780/commits/923a94ec15000dc13367e5324a028998bcab0ac8